packages/mux-player/test/player.test.js

```diff
@@ -548,6 +548,19 @@ describe('<mux-player>', () => {
         'https://image.mux.com/bos2bPV3qbFgpVPaQ900Xd5UcdM6WXTmz02WZSz01nJ00tY/storyboard.vtt?token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Ik96VU90ek1nUWhPbkk2MDJ6SlFQbU52THR4MDBnSjJqTlBxN0tTTzAxQlozelEifQ.eyJleHAiOjE5NjE2MDE3NzcsImF1ZCI6InMiLCJzdWIiOiJib3MyYlBWM3FiRmdwVlBhUTkwMFhkNVVjZE02V1hUbXowMldaU3owMW5KMDB0WSJ9.aVd0dsOJUVeQko3BWd9YEhL41Eytf_ZfaBeNzHSSUqU_gREa_jJEVTlRfuiE4g71cKJLSiVTKP7f-F7Txh6DlL8E2SkonfIPB2H0f_3DQxYLso2E8qI4zuJkyxKORbQFLAEB_vSE-2lMbrHXfdpQhv6SrVyu6di9ku0LpFpoyz-_7fVJICr8nhlsqOGt66AYcaa99TXoZ582FWzBaePmWw-WWKYsLvtNjLS9UoxbdVaBRwNylohvhh-i1Y9dNilyNooJ7O8Cj4GuMjeh1pCj0BOrGagxrWrswm3HjUVNUqFq5JCWnJCxgjjwiV4RLZg_4z7gkBXyX7H2-i1dKA3Cpw'
       );
     });
+    it('should work with storyboard-src', async function () {
+      const player = await fixture(`<mux-player
+          playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
+          stream-type="on-demand"
+          storyboard-src="https://image.mux.com/VcmKA6aqzIzlg3MayLJDnbF55kX00mds028Z65QxvBYaA/storyboard.vtt"
+          muted
+        ></mux-player>`);
+
+      assert.equal(
+        player.storyboard,
+        'https://image.mux.com/VcmKA6aqzIzlg3MayLJDnbF55kX00mds028Z65QxvBYaA/storyboard.vtt'
+      );
+    });
```

Comment:
todo(blocking): Add a few more tests here.

1. test setting the `storyboardSrc` property (will be very similar to this test)
2. test resetting `storyboardSrc` from an explicit URL to `undefined` (or `''`, depending on answer to above question)
3. test updating the `storyboardSrc` from one URL to another
4. test expected value when there is a `storyboard-token` applied (expectations/assertions for test will depend on question above re: `storyboard-token`

Reply:
I added a handful here; let me know if that's what you had in mind.